projects/js-packages/publicize-components/src/utils/get-share-message.js

```diff
@@ -0,0 +1,20 @@
+import { select } from '@wordpress/data';
+import { store as editorStore } from '@wordpress/editor';
+import { getShareMessageMaxLength } from './get-share-message-max-length';
+
+/**
+ * Gets the message that will be used hen sharing this post.
+ *
+ * @returns {string} The share message.
+ */
+export function getShareMessage() {
+	const { getEditedPostAttribute } = select( editorStore );
+	const meta = getEditedPostAttribute( 'meta' );
+	const message = `${ meta?.jetpack_publicize_message || '' }`;
```

Comment:
What benefit does this have over:

```suggestion
	const message = meta?.jetpack_publicize_message || '';
```

Is it that it ensures it's a string?

Reply:
Yes, just to ensure it's a string to allow auto completion.